pkgs/by-name/ba/bash-language-server/package.nix

```diff
@@ -60,6 +61,8 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru.tests.version = testers.testVersion { package = finalAttrs.finalPackage; };
```

Comment:
Is there a chance you could switch to [`versionCheckHook`](https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook) please?

Reply:
Yes, I could changed to your mentioned hook. Thanks, the hook is new for me.